### PR TITLE
Fat 4977

### PR DESCRIPTION
**Pull request by TetianaParanich**

### CODE REVIEW

File: cypress/e2e/data-import/e2e/match-on-pol-and-update-instance.cy.js [hunk 1]
```diff
@@ -23,11 +23,12 @@ import InventoryInstance from '../../../support/fragments/inventory/inventoryIns
 import InventoryViewSource from '../../../support/fragments/inventory/inventoryViewSource';
 import Users from '../../../support/fragments/users/users';
 import FileManager from '../../../support/utils/fileManager';
+import InventoryInstances from '../../../support/fragments/inventory/inventoryInstances';
 
 describe('ui-data-import', () => {
   let user = null;
   let orderNumber;
-  let instanceHrid;
+  const itemBarcode = '242451241241';
```
Comment (MaksimTerekhov) [inline cypress/e2e/data-import/e2e/match-on-pol-and-update-instance.cy.js:31]: why it is not a random number?) I assume better to use here is getRandomPostfix()

**Review by MaksimTerekhov — APPROVED** [review]

File: cypress/e2e/data-import/e2e/match-on-vrn-and-update-items.cy.js [hunk 2]
```diff
@@ -36,6 +37,7 @@ describe('ui-data-import', () => {
     quantityPhysical: '1',
     createInventory: 'Instance, holdings, item'
   };
+  const itemBarcode = 'xyzt124245271818912626262';
```
Comment (penfriend) [inline cypress/e2e/data-import/e2e/match-on-vrn-and-update-items.cy.js:40]: why don't you use here the random barcode?

File: cypress/e2e/data-import/e2e/match-on-pol-and-update-instance.cy.js (hunk 1 above)
Comment (penfriend) [inline cypress/e2e/data-import/e2e/match-on-pol-and-update-instance.cy.js:31]: why don't you use here the random barcode?

File: cypress/e2e/data-import/e2e/match-on-vrn-and-update-items.cy.js (hunk 2 above)
Comment (TetianaParanich) [reply cypress/e2e/data-import/e2e/match-on-vrn-and-update-items.cy.js]: Done

File: cypress/e2e/data-import/e2e/match-on-pol-and-update-instance.cy.js (hunk 1 above)
Comment (TetianaParanich) [reply cypress/e2e/data-import/e2e/match-on-pol-and-update-instance.cy.js:31]: Done

File: cypress/support/fragments/data_import/matchOnVRN.js [hunk 3]
```diff
@@ -113,7 +112,7 @@ const itemStatusKeyValue = KeyValue('Item status');
 const itemBarcodeKeyValue = KeyValue('Item barcode');
 const instanceDetailsSection = Section({ id: 'pane-instancedetails' });
 const viewSourceButton = Button('View source');
-const itemBarcodeLink = Link(itemBarcode);
+// const itemBarcodeLink = Link(itemBarcode);
```
Comment (IhorBohdan) [inline cypress/support/fragments/data_import/matchOnVRN.js:115]: Need here comment?

Comment (TetianaParanich) [reply cypress/support/fragments/data_import/matchOnVRN.js]: Deleted

**Review by IhorBohdan — APPROVED** [review]

**Pull request merged**